### PR TITLE
Fix #40: Default prompt choices title to value if not set?

### LINKED ISSUE #40: Default prompt choices title to value if not set?
Can submit a PR at some point if needed.

### PR DESCRIPTION
**Pull request by ianwalter**
Fixes #40. Changing multiselect choice value map function so that the choice title defaults to the choice value if not defined.

### CODE REVIEW

**Comment by terkelg** [comment]
Thanks @ianwalter. I agree with Luke, let's remove the unnecessary `{}`

File: lib/elements/multiselect.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ class MultiselectPrompt extends Prompt {
     this.cursor = cursor;
     this.hint = hint || '- Space to select. Return to submit';
     this.maxChoices = max;
-    this.value = choices.map(v => Object.assign({}, { selected: false }, v));
+    this.value = choices.map(v => {
+      return Object.assign({}, { title: v.value, selected: false }, v);
```
Comment (terkelg) [inline lib/elements/multiselect.js]: I know I wrote this too but let's remove the empty object for now

**Comment by ianwalter** [comment]
No problem. Just curious though, why arent we commiting the package-lock file?

**Comment by terkelg** [comment]
The quick and nonchalant answer: Because it's ugly. 
As a very visual person `package-lock` It makes my very sad and my eyes bleed. I know it's best practice ... but hey aesthetics.

**Comment by lukeed** [comment]
Hahah, it's also not very useful in this case. There are only two dependencies, so there is very very low risk of version mismatch. If I'm not mistaken, both of these dependencies also don't have any of their own, which means the lock file is even less useful.

**Comment by ianwalter** [comment]
Ok, I’ll add it to .gitignore then.

**Pull request merged**